main/management/commands/kafka_consumer.py

```diff
@@ -0,0 +1,51 @@
+"""Django management command to populate Kafka messages into application database."""
+
+from argparse import ArgumentParser
+from typing import Any
+
+from django.conf import settings
+from django.contrib.sessions.backends.db import SessionStore
+from django.contrib.sessions.models import Session
+from django.core.management.base import BaseCommand
+from django.db import transaction
+from druncschema.broadcast_pb2 import BroadcastMessage
+from kafka import KafkaConsumer
+
+
+class Command(BaseCommand):
+    """Consumes messages from Kafka and stores them in active user sessions."""
+
+    help = __doc__
+
+    def add_arguments(self, parser: ArgumentParser) -> None:
+        """Add commandline options."""
+        parser.add_argument("--debug", action="store_true")
+
+    def handle(self, debug: bool = False, **kwargs: Any) -> None:  # type: ignore[misc]
```

Comment:
Presumably this is because we disallow explicit `Any`? Tbh I'm not sure that setting is super helpful -- there are cases (like this one) where an explicit `Any` makes sense. Maybe we should change that?

Reply:
I actually like to disallow explicity Any as I think Any's can be a bit of a crutch so prefer that we think carefully about where we use them and explicitly silence the check where necessary.